router/account.py

```diff
@@ -13,7 +13,7 @@
 )
 
 @router.post(
-    "/create", tags=["create"], responses={
+    "/", tags=["create", "HTTP_POST"], responses={
```

Comment:
tag "create" and "HTTP_POST" is the same concept, remove one of them so swagger looks cleaner, the same thing applies to get/HTTP_GET and update/HTTP_PUT

Reply:
should be fixed in 973f9d6